src/SelfMeasurements-2-0/Cert/_reference/resources-query-send/medmij-selfmeasurements-HeartRate-kwalificatie5.xml

```diff
@@ -0,0 +1,50 @@
+<Observation xmlns="http://hl7.org/fhir" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hl7.org/fhir http://hl7.org/fhir/STU3/observation.xsd">
+    <id value="medmij-selfmeasurements-HeartRate-kwalificatie5"/>
+    <meta>
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-HeartRate"/>
+    </meta>
+    <status value="final"/>
+    <category>
+        <coding>
+            <system value="http://hl7.org/fhir/observation-category"/>
+            <code value="vital-signs"/>
+            <display value="Vital Signs"/>
+        </coding>
+    </category>
+    <code>
+        <coding>
+            <system value="http://loinc.org"/>
+            <code value="8867-4"/>
+            <display value="Heart rate"/>
+        </coding>
+    </code>
+    <subject>
+        <reference value="Patient/selfmeasurement-patient-kwalificatie2"/>
+        <display value="Marieke XXX_Bergzoon"/>
+    </subject>
+    <effectiveDateTime value="${DATE, T, D, -37}T08:15:00+02:00"/>
+    <performer>
+        <reference value="Patient/selfmeasurement-patient-kwalificatie2"/>
```

Comment:
Doen we niets met PractitionerRole?

Reply:
Nee, ik heb gekeken naar de andere resources (zoals bodyweight & bloodpressure) en hier is ook geen gebruik gemaakt van PractitionerRole. In het Addendum ontbreekt op dit moment nog de rol van de zorgverlener. Wellicht een nieuw ticket voor aanmaken?

Reply:
Ok, lijkt me voor nu wel oke.